sockpuppet/templatetags/sockpuppet.py

```diff
@@ -25,7 +26,19 @@ def render(self, context):
             elif node.token.token_type.name == 'VAR':
                 raw = '{{ ' + node.token.contents + ' }}'
             else:
-                msg ='{} is not yet handled'.format(node.token.token_type.name)
+                msg = '{} is not yet handled'.format(node.token.token_type.name)
                 raise Exception(msg)
             output = output + raw
         return output
+
+
+@register.simple_tag
+def stimulus(reflex, **kwargs):
```

Comment:
I agree with @DamnedScholar, reflex is a better name for this :).

Reply:
I dont mind at all so I'll take whatever you want :D